tools/chapel-py/src/core-types.cpp

```diff
@@ -502,6 +503,13 @@ PyObject* AstNodeObject_type(AstNodeObject *self, PyObject *Py_UNUSED(ignored))
   return ret;
 }
 
+PyObject* AstNodeObject_called_fn(AstNodeObject *self, PyObject *Py_UNUSED(ignored)) {
```

Comment:
METH_NOARGS means you don't need to put `PyObject *Py_UNUSED(ignored)` at all

Reply:
This is good to know and I've fixed the issue for all methods in AstNode (most of them had this problem).